pkg/oci/spec_opts_windows.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"golang.org/x/sys/windows"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
I think the `pkg/oci` package is mostly intended to be generic (not CRI-specific); wondering if the `WithAffinityCPUs` should have the CRI types in its signature, or if `internal/cri` needs a utility to convert to the equivalent OCI types.

Reply:
Maybe an equivalent in `internal/cri/opts` that does the conversion (which could be a wrapper around a generic variant in `pkg/oci`); https://github.com/containerd/containerd/blob/d6147c310461f4aac9f04cc75c5e99e3852fecef/internal/cri/opts/spec_windows.go#L42-L46

Reply:
had to move the function out to spec_opts.go to make linter happy